code/modules/mob/living/carbon/xenomorph/castes/drone/castedatum_drone.dm

```diff
@@ -42,10 +42,10 @@
 	caste_traits = list(TRAIT_CAN_VENTCRAWL)
 
 	// *** Defense *** //
-	soft_armor = list(MELEE = 10, BULLET = 10, LASER = 10, ENERGY = 10, BOMB = 0, BIO = 0, FIRE = 10, ACID = 0)
+	soft_armor = list(MELEE = 31, BULLET = 31, LASER = 31, ENERGY = 31, BOMB = 0, BIO = 15, FIRE = 31, ACID = 15)
```

Comment:
I know this is old stats but please make them all multiples of 5. 30 instead of 31, etc.